orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -455,3 +454,25 @@ def _initialize_graph_builder(self, training):
     def signal_model_changed(self):
         """Signals the execution manager to re-export the model on the next forward call"""
         self._original_model_has_changed = True
+
+    def __getstate__(self):
+        state = copy.copy(self.__dict__)
+        # Remove any re-contructible/pybound object from the state
```

Comment:
```suggestion
        # Remove any re-constructible/pybound object from the state
```

Reply:
Will get this in another PR along with other changes.